main/acle.md

```diff
@@ -4983,14 +4974,22 @@ function F if at least one of the following is true:
 Otherwise, ZA can be in any state on entry to A if at least one of the
 following is true:
 
-*   F [uses](#uses-state) `"za"`
+*   F [uses](#uses-state) `"za"`.
+
+*   F [uses](#uses-state) `"zt0"`.
+
+*   F is what the [[AAPCS64]](#AAPCS64) calls an ["agnostic-ZA"](#agnostic-za)
+    function and A's clobber-list does not include `"za"` or `"zt0"`.
+
+Otherwise, ZA can be off or dormant on entry to A if at least one of the
```

Comment:
I think we should just keep the existing “Otherwise…” paragraph as-is.  AFAIK, there are no “otherwise” cases in which ZA is guaranteed to be off on entry.